service/provider/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
   implementation "org.springframework.boot:spring-boot-starter-data-redis"
   implementation "org.apache.camel.springboot:camel-spring-redis-starter:${camel_version}"
   implementation "org.apache.camel.springboot:camel-jsonpath-starter:${camel_version}"
+  implementation "org.apache.camel.springboot:camel-bean-validator-starter:${camel_version}"
```

Comment:
validate beans as part of the route

Reply:
This has introduced a vulnerability -- issue #735 created.

Reply:
This must be the straw that broke the camel's back :) We don't necessarily need this library. We can perform validation by other means. Let me know if you'd like me to remove it.

Reply:
Yes, let's replace it. The alert states:
> org.glassfish:javax.el is deprecated, users can move to use org.glassfish:jakarta.el instead where this issue is first fixed in version 3.0.4.